www/caddy/src/opnsense/mvc/app/controllers/OPNsense/Caddy/forms/dialogAccessList.xml

```diff
@@ -23,13 +23,15 @@
         <id>accesslist.HttpResponseCode</id>
         <label>HTTP Response Code</label>
         <type>text</type>
-        <help><![CDATA[Set a custom HTTP response code that should be returned to the requesting client when the access list does not match. Setting this will replace "Abort Connections" only for this access list. All clients will stay connected but will receive the response code.]]></help>
+        <help><![CDATA[Set a custom HTTP response code that should be returned to the requesting client when the access list does not match. If empty, the connection is aborted with no response.]]></help>
```

Comment:
Reading that again is that the default caddy behaviour too?

Reply:
There is no default behavior here. I am constructing that default behavior out of best practices.

For me, `abort` is the default behavior since I write it into the config.

The HTTP Code and Response is for some edge use cases and was introduced here. It does not have to be filled out by the user, and does not influence if the access list aborts connections, it just alters the response:

https://github.com/opnsense/plugins/issues/3883